src/stylesheets/themes/hugo-terminal-orange/placeholder.scss

```diff
@@ -0,0 +1,13 @@
+@import "./variables";
+
+@mixin placeholder {
+  ::placeholder { @content; }
+  ::-webkit-input-placeholder { @content; }
+  ::-moz-placeholder { @content; }
+  :-moz-placeholder { @content; }
+  :-ms-input-placeholder { @content; }
```

Comment:
I think you can remove the prefixed versions of `::placeholder`. Autoprefixer will add them as part of the scss build.